x-pack/plugins/secops/public/containers/source/index.tsx

```diff
@@ -43,3 +44,6 @@ export const WithSource = ({ children, sourceId }: WithSourceProps) => (
     }}
   </Query>
 );
+
+export const indiciesExistOrDataTemporarilyUnavailable = (indiciesExist: boolean) =>
```

Comment:
This type should be: `indiciesExist: boolean | undefined` no?

Reply:
You're right, it probably makes more sense to explicitly call out it can be `undefined` as well -- I'll make the change. I was basing it off of the [type of `auditbeatIndicesExist`](https://github.com/elastic/kibana/blob/feature-secops/x-pack/plugins/secops/public/containers/source/index.tsx#L16) from the `WithSource` component, which is just a `boolean`.